quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.unix.approved.txt

```diff
@@ -150,14 +150,17 @@ HTTP(S):
 --https-protocols <protocols>
                      The list of protocols to explicitly enable. Default: TLSv1.3,TLSv1.2.
 --https-trust-store-file <file>
-                     DEPRECATED: The trust store which holds the certificate information of the
-                       certificates to trust.
+                     DEPRECATED. Use the System Truststore instead, see the docs for details. --
```

Comment:
Is it expected the `--` here?

Reply:
Rewrote the message a bit. Now it looks like
<img width="876" alt="Screenshot 2023-12-11 at 15 08 00" src="https://github.com/keycloak/keycloak/assets/13906855/66a3cfab-fbe8-46c3-96fd-25e5b7321340">
<img width="876" alt="Screenshot 2023-12-11 at 14 37 03" src="https://github.com/keycloak/keycloak/assets/13906855/a89469af-c38e-49f2-ba1d-c08beeb4e9b7">